src/components/TimeseriesDataExport/TimeseriesDataExport.tsx

```diff
@@ -118,11 +87,65 @@ const TimeseriesDataExportFC: FC<TimeseriesDataExportFormProps> = (
     imageDownloadBtn,
   } = lang;
 
+  const getLimits = async (request: DatapointsMultiQuery) => {
+    let { start = 0, end = 0 } = request;
+
+    const project = context!.project || '';
+    const url = `/api/v1/projects/${project}/timeseries/data/latest`;
+    // @ts-ignore
+    const items = request.items.map(({ id }) => ({ id }));
+
+    const endResults = await context!.post<{ items: any[] }>(url, {
+      data: { items },
+    });
+    end = Math.min(
+      ...endResults.data.items.map(item => +item.datapoints[0].timestamp, end)
+    );
+
+    const getFirstDatapointRequest = {
+      ...request,
+      start,
+      end,
+      limit: CELL_LIMIT,
+    };
+
+    const startResult = await context!.datapoints.retrieve(
+      getFirstDatapointRequest
+    );
+    start = Math.max(
+      // @ts-ignore
```

Comment:
ignore should disappear after using sdk endpoint instead

Reply:
Even after switching I'm still getting type-errors.  `Property id does not exist on type  DatapointsQuery`, which it very much does.  Could the issue be something similar to: https://github.com/cognitedata/gearbox.js/pull/646?